backend/views/error.jade

```diff
@@ -0,0 +1,6 @@
+extends layout
```

Comment:
Do we really need views?

Reply:
I'll remove them

Reply:
It gave an error when I removed the views so I had to add them back